hw/ip/aes/lint/aes.waiver

```diff
@@ -3,7 +3,7 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 # waiver file for aes
-waive -rules COMBO_READ_WRITE -location {aes_key_expand.sv} -regexp {'regular\[[0-9]+\]\[2:1|4:2|6:5\]' is read from within the same combinational process block} \
+waive -rules COMBO_READ_WRITE -location {aes_key_expand.sv} -regexp {'regular\[[0-9]+:?[0-9]*\]\[2:1|4:2|6:5\]' is read from within the same combinational process block} \
```

Comment:
```suggestion
waive -rules COMBO_READ_WRITE -location {aes_key_expand.sv} -regexp {'regular\[\d+:?\d*\]\[2:1|4:2|6:5\]' is read from within the same combinational process block} \
```

Reply:
But do you really have to make the waiver that strict?

Reply:
Thanks for your feedback @imphil . Combo loops can be really bad. Thus I think it's better to have some false alarms compared to letting a real loop slip.

Reply:
I guess I missed that. Ascentlint is very good at finding combo loops, so avoid masking that detection.